src/clients/firewall_auto.rb

```diff
@@ -87,9 +87,7 @@ def main
         SuSEFirewall.SetStartService(SuSEFirewall.GetEnableService)
       # Import configuration
       elsif @func == "Import"
-        @ret = SuSEFirewall.Import(
-          Convert.convert(@param, :from => "map", :to => "map <string, any>")
-        )
+        @ret = SuSEFirewall.read_and_import(@param)
```

Comment:
and btw when I see usage, then maybe it is better to modify import to have by default read and import. E.G. bootloader is done in this way